app/components/user_programme_course_bookings_with_asides_component.rb

```diff
@@ -28,15 +28,17 @@ def course_sections
   def in_progress_courses
     @current_user.achievements
       .in_state(:in_progress, :enrolled)
-      .with_courses.order("created_at DESC")
+      .with_courses.order("achievements.created_at DESC")
```

Comment:
@A-Wheeto I added includes on here to clear a bullet issue. I assumed it was the achievements you were ordering by?

Reply:
Thanks Mike, yes that should be achievements that are ordered.